### PR TITLE
Notification page changes

### PR DESCRIPTION
**Pull request by hellypi**

### CODE REVIEW

**Comment by hellypi** [comment]
[Notification page changes](https://trello.com/c/tCQbVo1H/516-notification-page-changes)

**Comment by hellypi** [comment]
We'll have to cherry-pick the commits later, starting with 
@csrapr Feat: add notification center row in notification settings table 
& onwards

File: lib/notifier/index.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+function transactionNotify (tx, rec) {
+  return settingsLoader.loadLatest().then(settings => {
+    const notifSettings = configManager.getGlobalNotifications(settings.config)
+    const highValueTx = tx.fiat.gt(notifSettings.highValueTransaction || Infinity)
+    const isCashOut = tx.direction === 'cashOut'
+
+    // for notification center
+    const directionDisplay = tx.direction === 'cashOut' ? 'cash-out' : 'cash-in'
+    const readyToNotify = tx.direction === 'cashIn' || (tx.direction === 'cashOut' && rec.isRedemption)
```
Comment (chaotixkilla) [inline lib/notifier/index.js]: Why not reuse the variable `isCashOut`?
```suggestion
    const isCashOut = tx.direction === 'cashOut'

    // for notification center
    const directionDisplay = isCashOut ? 'cash-out' : 'cash-in'
    const readyToNotify = !isCashOut || (isCashOut && rec.isRedemption)
```

File: new-lamassu-admin/src/styling/icons/action/wrench/zodiac.svg [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        <g id="notification-center_v01a#2-(open)" transform="translate(-1023.000000, -459.000000)" stroke="#1B2559">
+            <g id="Group-5" transform="translate(1000.000000, 0.000000)">
+                <g id="icon/sf-small/wrench" transform="translate(24.000000, 460.000000)">
+                    <path d="M15.7602493,3.10720971 L13.1962412,5.67121772 L10.3290323,5.67121772 L10.3290323,2.80400876 L12.8930403,0.24000075 C12.4378389,0.0872002725 11.9506373,0 11.4434358,0 C8.9282279,0 6.88822153,2.04000637 6.88822153,4.55681424 C6.88822153,5.08081588 6.98102182,5.58321745 7.14422233,6.05201891 L0.580201813,12.6168394 C-0.193400604,13.3904418 -0.193400604,14.6456458 0.580201813,15.4200482 C1.35460423,16.1936506 2.60980816,16.1936506 3.38341057,15.4200482 L9.94823109,8.85602767 C10.4170326,9.01922818 10.9186341,9.11202847 11.4434358,9.11202847 C13.9602436,9.11202847 16.00025,7.0720221 16.00025,4.55681424 C16.00025,4.04961265 15.9130497,3.56241113 15.7602493,3.10720971 Z" id="Stroke-1"></path>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/styling/icons/action/wrench/zodiac.svg]: Not sure if supposed, but .svg files are missing the newline at the end

**Review by chaotixkilla — CHANGES_REQUESTED** [review]
Some very minor stuff. There are some comments throughout the code, I'm not sure which ones were temporary and which were to inform future people working on the code, so I marked none.

Conflicts need to be fixed as well

**Comment by hellypi** [comment]
Again, first commit that "counts" is:
@csrapr
Feat: add notification center row in notification settings table

File: lib/notifier/queries.js [hunk 3]
```diff
@@ -14,75 +14,82 @@ compliance - notifications related to warnings triggered by compliance settings
...
+function getMachineName (machineId) {
+  const sql = 'SELECT * FROM devices WHERE device_id=$1'
+  return db.oneOrNone(sql, [machineId])
+    .then(it => it.name)
+}
+
 const addNotification = (type, message, detail) => {
-    const sql = `INSERT INTO notifications (id, type, message, detail) values ($1, $2, $3, $4)`
-    return db.oneOrNone(sql, [uuidv4(), type, message, detail])
+  const sql = `INSERT INTO notifications (id, type, message, detail) values ($1, $2, $3, $4)`
```
Comment (chaotixkilla) [inline lib/notifier/queries.js]: `values` should be capitalized, but it is not a problem
```suggestion
  const sql = `INSERT INTO notifications (id, type, message, detail) VALUES ($1, $2, $3, $4)`
```

**Review by chaotixkilla — APPROVED** [review]

File: lib/machine-loader.js [hunk 4]
```diff
@@ -3,7 +3,8 @@ const axios = require('axios')
 
 const db = require('./db')
 const pairing = require('./pairing')
-const notifier = require('./notifier')
+const checkPings = require('./notifier').checkPings
+const checkStuckScreen = require('./notifier').checkStuckScreen
```
Comment (RafaelTaranto) [inline lib/machine-loader.js]: Hmm, I'm not a fan of this style of import. Can't you do

```js
const {checkPings, checkStuckScreen} = require('./notifier')
```

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/machine-loader.js (hunk 4 above)
Comment (RafaelTaranto) [reply lib/machine-loader.js]: Also all of the svg changes are unneeded.

**Review by RafaelTaranto — APPROVED** [review]

File: lib/blacklist.js [hunk 5]
```diff
@@ -15,7 +15,7 @@ const getBlacklist = () => {
 // Delete row from blacklist table by crypto code and address
 const deleteFromBlacklist = (cryptoCode, address) => {
   const sql = `DELETE FROM blacklist WHERE crypto_code = $1 AND address = $2`
-  notifier.clearBlacklistNotification(cryptoCode, address)
+  notifierQueries.clearBlacklistNotification(cryptoCode, address).catch(console.error)
```
Comment (joshmh) [inline lib/blacklist.js]: Are we ever doing something with these errors other than printing to log? If not, ``notifierQueries`` should take care of it.

**Review by joshmh — CHANGES_REQUESTED** [review]

**Pull request merged**